HULKs/hulk
# ball detected far from kick off depending on seen location

## Why? What?

We have often had an issue where camera matrix miscalibrations lead to the robot seeing the ball at a location that was not the midpoint of the center circle, thus ```ball_detected_far_from_kick_off_point``` see #861.
This should fix that.

The cycle we hear the whistle we now save the location of the ball. We check for whether the ball has moved according to this seen location, if thats not an option we use our old method.

Additionally we don't use active vision during kickoff when the ball is not free. This means we look at the ball until either the gamecontroller says it is free or we see it move and it is free. This if the camera matrix is off we don't get random ball drifts due to head movements.

Fixes #861
Fixes #1043

## ToDo / Known Issues

## Ideas for Next Iterations (Not This PR)

Implementing something similar for the penalty shot direction provider

## How to Test

Take a robot and purposely set its camera matrix wrong until it always violates the center circle during kickoff using the old software. Now use the new software and see the difference.
